scripts/install.mjs

```diff
@@ -0,0 +1,121 @@
+// Install any non-NPM dependencies in this script.
```

Comment:
I get how this works locally, and successfully ran it. But I don't quite understand how this works in production, when gui is being included as a static library?

Reply:
Hmm... you have a point. I should double-check whether that works as expected.

Reply:
I moved this from `install` to `prepublish` based on the "Use Cases" section of the "Prepare and Prepublish" docs here: https://docs.npmjs.com/cli/using-npm/scripts, plus some experimentation. Thanks for catching this!

Reply:
Sounds good! We should check on staging that this works as expected once it's merged and up there -- I want to make sure fastly passes the request through correctly, and doesn't need its settings revised.